data/kubemacpool/kubemacpool.yaml

```diff
@@ -3,16 +3,15 @@ kind: Namespace
 metadata:
   labels:
     control-plane: mac-controller-manager
-    controller-tools.k8s.io: "1.0"
-    openshift.io/run-level: "0"
-    runlevel: "0"
+    mutatepods.kubemacpool.io: ignore
   name: {{ .Namespace }}
 ---
 apiVersion: admissionregistration.k8s.io/v1beta1
 kind: MutatingWebhookConfiguration
 metadata:
   labels: null
   name: kubemacpool-mutator
+  namespace: '{{ .Namespace }}'
```

Comment:
this is not present in the original manifest of kmp. 
I guess it's because of differences in kustomize version, and I don't see any harm in it.